docs/setup/independent/install-kubeadm.md

```diff
@@ -193,6 +174,10 @@ kubeadm to tell it what to do.
 
 {% endcapture %}
 
+## Troubleshooting
+
+If you are running into difficulties with kubeadm, please consult our [troubleshooting docs](/docs/setup/independent/troubleshooting-kubeadm/).
+
```

Comment:
When previewing this doc in netlify, the section `Troubleshooting` is at the beginning of the doc. It's incorrect. Please move L177-180 ahead of L175 `{% endcapture %}`.